Source/Core/DolphinQt/GCMemcardManager.cpp

```diff
@@ -78,11 +98,20 @@ void GCMemcardManager::CreateWidgets()
   // Actions
   m_select_button = new QPushButton;
   m_copy_button = new QPushButton;
-
-  // Contents will be set by their appropriate functions
   m_delete_button = new QPushButton(tr("&Delete"));
-  m_export_button = new QPushButton(tr("&Export..."));
-  m_export_all_button = new QPushButton(tr("Export &All..."));
+
+  m_export_button = new QToolButton(this);
+  m_export_menu = new QMenu(m_export_button);
+  m_export_gci_action = new QAction(tr("&Export as .gci..."), m_export_menu);
+  m_export_gcs_action = new QAction(tr("Export as .&gcs..."), m_export_menu);
+  m_export_sav_action = new QAction(tr("Export as .&sav..."), m_export_menu);
+  m_export_menu->addAction(m_export_gci_action);
+  m_export_menu->addAction(m_export_gcs_action);
+  m_export_menu->addAction(m_export_sav_action);
+  m_export_button->setText(tr("&Export as .gci..."));
```

Comment:
Have you looked into using `m_export_button->setDefaultAction(m_export_gci_action)` instead? Usually with dropdown buttons, reusing actions from the menu should be preferred to avoid duplication. [According to documentation](https://doc.qt.io/qt-5/qtoolbutton.html#setDefaultAction), it sets the text amongst other properties.

Reply:
Ah, no I have not, I'll check if that works.

Reply:
Yup that works, did that.